lib/remote_persistent_term/fetcher/s3.ex

```diff
@@ -30,11 +30,12 @@ defmodule RemotePersistentTerm.Fetcher.S3 do
       required: true,
       doc: "The AWS region of the s3 bucket."
     ],
-    failover_regions: [
-      type: {:list, :string},
+    failover_buckets: [
+      type: {:list, :keyword_list},
       required: false,
       doc:
-        "A list of AWS regions to use if calls to the default region fail. They will be tried in order."
+        "A list of keyword lists containing [bucket: bucket_name, region: region] to use as failover if the primary bucket fails. \n
```

Comment:
if you define the schema, nimble options should automatically add some docs about the types, so you can just leave info about what it is used for